vaccine-standby/.env

```diff
@@ -0,0 +1,19 @@
+# description: Your Twilio phone number for sending and receiving messages
+# format: phone_number
+# required: true
+TWILIO_PHONE_NUMBER=
+
+# description: Use this password to access your intake form results
```

Comment:
We've seen confusion around these things in the past so let's make the description a bit more specific. 

```suggestion
# description: Set a password for your app. Users who want to configure the app or view who messaged in, will need this password.
```